It looks like golint is failing: 

```
## ⚠ golint Failed Found 2 lint suggestions; failing. <details><summary>Show Detail</summary> ``` core/move/move.go:45:6: func name will be used as move.MoveFromSGFPoint by other packages, and that stutters; consider calling this FromSGFPoint core/move/move.go:59:6: func name will be used as move.MoveListFromSGFPoints by other packages, and that stutters; consider calling this ListFromSGFPoints ``` </details>
```

--- a/core/move/move.go
+++ b/core/move/move.go
@@ -40,7 +40,7 @@ func (m *Move) IsPass() bool {
 	return m.point == nil
 }
 
-// MoveFromSGFPoint converts from an SGF point of the form "ab" to a point
+// FromSGFPoint converts from an SGF point of the form "ab" to a point
 // object, such as {0,1}.
 func FromSGFPoint(col color.Color, sgfPt string) (*Move, error) {
 	if sgfPt == "" {
@@ -54,7 +54,7 @@ func FromSGFPoint(col color.Color, sgfPt string) (*Move, error) {
 	return &Move{color: col, point: pt}, nil
 }
 
-// MoveListFromSGFPoints a move list of the form "ab", "bc" to a moves of the form
+// ListFromSGFPoints a move list of the form "ab", "bc" to a moves of the form
 // {0,1}, {0,2}. Note that pass-moves are not allowed in move-lists.
 func ListFromSGFPoints(col color.Color, sgfPts []string) ([]*Move, error) {
 	var moves []*Move

